src/Build/Resources/Strings.resx

```diff
@@ -1225,7 +1225,7 @@
     <comment>{StrBegin="MSB4091: "}</comment>
   </data>
   <data name="UnexpectedTaskAttribute" xml:space="preserve">
-    <value>MSB4064: The "{0}" parameter is not supported by the "{1}" task. Verify the parameter exists on the task, and it is a settable public instance property.</value>
+    <value>MSB4064: The "{0}" parameter is not supported by the "{1}" task loaded from assembly: {2} from the path: {3}. Verify that the parameter exists on the task, the UsingTask points to the correct assembly and it is a settable public instance property.</value>
```

Comment:
super-nit: In many other error messages mentioning `UsingTask` I see it wrapped in angle brackets making it clear that it refers to an XML element. Also, a comma before the last `and` would be more consistent with the style of the rest of this file.